eve/service/svc_note.py

```diff
@@ -0,0 +1,172 @@
+import json
+from rich.console import Console
+from rich.table import Table
+from rich import box
+import os.path
+from datetime import date
+import pathlib
+
+
+class Note:
+    def __init__(self):
+        # path to JSON-file where to store note-data
+        self.path = str(pathlib.Path.home()) + '/test.json'  # rename for production and make it a hidden file
+
+        # check if JSON-file already exists and if not create it with sample entries
+        if os.path.isfile(self.path):
+            pass
+        else:
+            self.touch = '''[
+                {
+                    "UID": 1,
+                    "CreationDate": "2021-04-24",
+                    "AlterationDate": null,
+                    "DueDate": "2021-04-30",
+                    "Priority": null,
+                    "NoteTxt": "This is sample entry 1."
+                },
+                {
+                    "UID": 2,
+                    "CreationDate": "2021-04-24",
+                    "AlterationDate": "2021-04-24",
+                    "DueDate": "2021-04-29",
+                    "Priority": 1,
+                    "NoteTxt": "This is sample entry 2."
+                },
+                {
+                    "UID": 3,
+                    "CreationDate": "2021-04-24",
+                    "AlterationDate": "2021-04-24",
+                    "DueDate": "2021-04-28",
+                    "Priority": 2,
+                    "NoteTxt": "This is sample entry 3."
+                }
+            ]'''
```

Comment:
It might be easier to store your notes as a single `dict`:

Like this, then you can look up the notes by id rather than having to iterate through the list every time to find the UID you want to edit?
```python
{
  'uuid123': {'creationDate': '', 'DueDate': ''}
  'uuid456': {'creationDate': '', 'DueDate': ''}
  'uuid789': {'creationDate': '', 'DueDate': ''}
}
```

Reply:
Yeah, might be easier, but I struggle a bit at the moment because dictionaries are unordered and I want to order the uids for output by DueDate and Priority.

Reply:
Ok, stackoverflow solved the problem :-)